pkg/controller/seed-controller-manager/initial-application-installation-controller/doc.go

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2022 The Kubermatic Kubernetes Platform contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+/*
+Package initialapplicationinstallationcontroller contains a controller that watches Cluster
+resources and will transform the InitialApplicationInstallations annotation into
+a list of ApplicationInstallation resource and create those resources in the
+user cluster, once the cluster has become ready.
```

Comment:
> contains a controller that watches Cluster
resources, specifically the InitialApplicationInstallations annotation. 

It can't "watch" an annotation. I'll reword this in a follow up PR